docs/ad/settings.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Settings
+parent: Anomaly Detection
+nav_order: 4
+---
+
+# Settings
+
+The anomaly detection plugin adds several settings to the standard Elasticsearch cluster settings.
+They are dynamic, so you can change the default behavior of the plugin without restarting your cluster.
+You can mark them `persistent` or `transient`.
+
+To update the retention period of the result index:
+
+```json
+PUT _cluster/settings
+{
+  "transient": {
+    "opendistro.anomaly_detection.ad_result_history_retention_period": "5m"
+  }
+}
+```
+
+Setting | Default | Description
+:--- | :--- | :---
+`opendistro.anomaly_detection.enabled` | True | Whether the anomaly detection plugin is enabled or not. If disabled, all detectors immediately stop running.
+`opendistro.anomaly_detection.max_anomaly_detectors` | 1,000 | The maximum number of non-high cardinality detectors (no category field) users can create.
+`opendistro.anomaly_detection.max_multi_entity_anomaly_detectors` | 10 | The maximum number of non-high cardinality detectors (no category field) in a cluster.
+`opendistro.anomaly_detection.max_anomaly_features` | 5 | The maximum number of features for a detector.
+`opendistro.anomaly_detection.ad_result_history_rollover_period` | 12h | How often the rollover condition is checked. If `true`, the plugin rolls over the result index to a new index.
+`opendistro.anomaly_detection.ad_result_history_max_docs` | 250000000 | The maximum number of documents in the result index. The plugin only counts refreshed documents in the primary shards.
```

Comment:
Since there can be multiple result indices, can we change to "The maximum number of documents in one result index"?